assets/javascripts/discourse/components/chat-message.js

```diff
@@ -563,6 +563,10 @@ export default Component.extend({
     this._loadingReactions.push(emoji);
     this._updateReactionsList(emoji, reactAction, this.currentUser);
 
+    if (reactAction === this.ADD_REACTION) {
+      this.emojiReactionStore.track(`:${emoji}:`);
```

Comment:
Should we rename the service here to match other usages?

Reply:
oh yes indeed forgot this one :+1: thanks